bin/start_swagger_ui

```diff
@@ -2,14 +2,19 @@
 
 set -euo pipefail
 
+source bin/util
+
 docker pull swaggerapi/swagger-editor
 
-echo "Removing old containers..."
+announce "Removing old containers..."
 docker stop swagger-editor || true
 docker rm swagger-editor || true
 
-echo "Starting Swagger editor on port 9090..."
-docker run -d -p 9090:8080 -v $(pwd):/tmp -e SWAGGER_FILE=/tmp/spec/openapi.yml --name swagger-editor swaggerapi/swagger-editor
+announce "Generating bundled spec file"
+./bin/bundle_spec
+
+announce "Starting Swagger editor on port 9090..."
+docker run -d -p 9090:8080 -v $(pwd):/tmp -e SWAGGER_FILE=/tmp/spec.yml --name swagger-editor swaggerapi/swagger-editor
```

Comment:
I don't know if you experience this as well, but the editor doesn't open to the file defined at SWAGGER_FILE consistently. Usually opens to whichever file I've opened last.

Reply:
I haven't seen this issue before, but I also haven't really used the editor that much. Perhaps its because the container doesn't properly shut down between each time you open it? I can't imagine that type of persistence occurring across completely recreated containers.